keras/src/layers/core/dense.py

```diff
@@ -661,7 +778,12 @@ def quantize(self, mode, type_check=True):
         if self.dtype_policy.quantization_mode is None:
             from keras.src import dtype_policies  # local import to avoid cycle
 
-            policy = dtype_policies.get(f"{mode}_from_{self.dtype_policy.name}")
+            policy_name = mode
+            if mode == "gptq":
+                policy_name = config.dtype_policy_string()
+            policy = dtype_policies.get(
+                f"{policy_name}_from_{self.dtype_policy.name}"
```

Comment:
That will give you `gptq_from_...`. Don't you need the other parameters? `qptq/.../..._from_...`?